go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -126,6 +127,12 @@ type Throttler struct {
 	srvTopoServer   srvtopo.Server
 	heartbeatWriter heartbeat.HeartbeatWriter
 
+	// recentCheckTickerValue is an ever increasing number, incrementing once per second.
+	recentCheckTickerValue int64
+	// recentCheckValue is set to match or exceed recentCheckTickerValue whenever a "check" was made (other than by the throttler itself).
+	// when recentCheckValue < recentCheckTickerValue that means there hasn't been a recent check.
```

Comment:
IMO 1 second ago is recent.

Reply:
Agreed, and that's what the code considers as "recent". `1s`. It's just that we have a 1 second granularity, and so the actual range can go as high as `1.999`.